src/applications/simple-forms/21-4138/containers/IntroductionPage.jsx

```diff
@@ -37,14 +59,35 @@ const IntroductionPage = props => {
   };
 
   return (
-    <IntroductionPageView
-      route={route}
-      content={content}
-      ombInfo={ombInfo}
-      childContent={childContent}
-      userIdVerified={userIdVerified}
-      userLoggedIn={userLoggedIn}
-    />
+    <>
+      <VaBreadcrumbs
```

Comment:
I think we don't include this. Breadcrumbs don't show up on localhost but they do on staging/production, so this would duplicate the breadcrumbs I believe. breadcrumbs were managed in `content-build`, for static site content

Reply:
Good catch, fixed! 👍